seqr/views/utils/orm_to_json_utils.py

```diff
@@ -760,7 +760,8 @@ def get_json_for_matchmaker_submission(submission):
 def get_json_for_rna_seq_outliers(models, **kwargs):
     additional_values = {
         'isSignificant': Case(
-            When(p_adjust__lt=models.model.SIGNIFICANCE_THRESHOLD, then=Value(True)), default=Value(False)
+            When(then=Value(True), **{f'{models.model.SIGNIFICANCE_FIELD}__lt': models.model.SIGNIFICANCE_THRESHOLD}),
```

Comment:
for splice junction outliers only the top 50 are significant, in addition to needing to be over the threshold

Reply:
Okay, I thought the 50 was for the limit of the outlier table.